src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Startlist/TabView.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NTS.Judge.Blazor.Pages.Dashboard.Startlist;
+public class TabView
```

Comment:
Don't like the naming, we're not using Views atm. 

Furthermore is this the best place for a generic component model?